app/src/main/res/values-da/strings.xml

```diff
@@ -63,6 +63,8 @@
     <string name="open_dialpad_by_default">Åben det numeriske tastatur som standard når appen åbner</string>
     <string name="disable_proximity_sensor">Deaktivér nærhedssensor under opkald</string>
     <string name="disable_swipe_to_answer">Erstat swipe ved besvarelse af indgående opkald med at klikke på</string>
+    <string name="hashtag">Hashtag</string>
+    <string name="app_launcher_name">Telefon</string>
     <string name="show_incoming_calls_full_screen">Vis altid indkomne opkald i fuld skærm</string>
```

Comment:
@lianergoist 

Why are the strings being duplicated? `hashtag` was already declared at line 17. I don't remember which one but more strings were duplicated by your Weblate commits in another Weblate PR.

Thanks.

Reply:
I am sorry. I didn't knew about weblate when I committed the first translation. Later, I used weblate, and that may have caused problems? In the future I will use weblate only.